### PR TITLE
TIP-657: short path to working CI

### PR DESCRIPTION
**Pull request by jjanvier**
This PR:
- allows to install the PIM with the icecat catalog
- allows to install the PIM with the Behat catalog
- allows to use the product grid
- allows to find products via their label in the product grid (useful for a lot of Behat)
- is a first short path to a working CI: we need that to be able to launch the Behats of the features we will work on (variant group, completeness, unique values, variant groups etc...)

For that, we disabled a couple of things in the PIM:
- the ACME Bundle that allows us to test the reference data 
- the unique values validation
- the check of the eligible products to a variant

=> all these things will be re-enabled and cleaned soon in the coming PRs :)

Also, this PR removes some code that become obsolete with the new storage of the values:
- the database collector that gathered information about the numder of product values (to know if Mongo should be activated or not)
- some product repository methods that joined on the values table

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | -
| Changelog updated                 | OK
| Review and 2 GTM                  | 
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/Repository/ProductValueCounterRepository.php [hunk 1]
```diff
@@ -13,7 +12,7 @@
  * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
-class ProductValueCounterRepository implements ProductValueCounterRepositoryInterface
+class ProductValueCounterRepository
```
Comment (damien-carcel) [inline src/Pim/Bundle/CatalogBundle/Doctrine/MongoDBODM/Repository/ProductValueCounterRepository.php]: No need for this repository if you remove the collector, I think.

File: src/Pim/Bundle/VersioningBundle/spec/Normalizer/Flat/ProductNormalizerSpec.php [hunk 2]
```diff
@@ -73,10 +73,10 @@ function it_normalizes_product(
 
         $this->normalize($product, 'flat', [])->shouldReturn(
             [
-                'sku'        => 'sku-001',
```
Comment (damien-carcel) [inline src/Pim/Bundle/VersioningBundle/spec/Normalizer/Flat/ProductNormalizerSpec.php]: I did this too (already merged on TIP-613), but in fact it is wrong.
We broke a PIM feature: having the sku first. You can see it on exports, the sku is in the middle instead being on the first column.
(but the fact is, I have no idea why it is this way…)

Comment (jjanvier) [reply src/Pim/Bundle/VersioningBundle/spec/Normalizer/Flat/ProductNormalizerSpec.php]: thanks I will dig on that :+1:

**Review by damien-carcel — APPROVED** [review]

**Review by SamirBoulil — APPROVED** [review]
GTM did you try to run a CI ?

**Comment by jjanvier** [comment]
@SamirBoulil no, but I ran some Behat on local
the one about the product imports

File: src/Pim/Bundle/VersioningBundle/spec/Normalizer/Flat/ProductNormalizerSpec.php (hunk 2 above)
Comment (jjanvier) [reply src/Pim/Bundle/VersioningBundle/spec/Normalizer/Flat/ProductNormalizerSpec.php]: in the export, the array converter are used
for the versioning, we don't care about the order

**Pull request merged**